### DISCUSSION TITLE
2022.10.15 브리핑

### DISCUSSION THREAD

**Discussion by itmakesmesoft** [itmakesmesoft/Steady-Study#123] [Self-Assessment]
이번 주 목표를 이루기 위해 어떤 노력을 했는지 스스로 평가해보고, 점수를 매겨주세요.
ex)
이번 주 목표 : 기술 블로그에 배운 내용 정리하기

점수 : 80점
이유 : (생략)

**Comment by itmakesmesoft** [comment]
## 이번 주 목표 및 달성률(2022.10.09 ~ 2022.10.15)
이은혁 : 자바 강의 객체지향 1까지 듣기(제발)
고영석 : IM 통과하기, 인적성 공부하기, 진도 따라가기, 블로그 글 3개 이상 올리기
김지애 : cs(DB)공부, DFS/BFS 완벽복습, 싸피 진도 따라가기
김지오 : IM통과, 울지않기😅, swea IM문제 다 풀고 블로그 올리기
나혜승 : IM 대비 오답노트 블로그에 정리하기,수업 진도 놓치지 않기
이수연 : Django 복습 CRUD 2번 반복하기, 1일 1알고리즘 문제 풀이(pass 못해도 제출이라도 하기), 자바 강의 5강 이상 수강
이문삼 : 연휴에 리액트 완강하기, cs(운영체제, 컨테이너) 공부하기, 수업 진도
정혜수 : Django 총 복습, DFS/BFS 등 알고리즘 복습, 수업 과제 챙기기
김준영 : A형 통과하기

**Comment by sooyeonlee127** [comment]
이번 주 목표: Django 복습 CRUD 2번 반복하기, 1일 1알고리즘 문제 풀이(pass 못해도 제출이라도 하기), 자바 강의 5강 이상 수강

점수: 30점
이유: 따로 Django 복습은 한 번밖에 못함 .. 1일 1알고리즘 문제 풀지 못함.. 자바 강의 5강 이상 수강함
이번 주 공부에 집중을 잘 못하고 수업에도 불성실하게 참여했던 것 같아서 반성합니다....
초심 다시 잡아오겠습니다... ㅜㅜ

**Comment by moonthree** [comment]
이번 주 목표 :  연휴에 리액트 완강하기, cs(운영체제, 컨테이너) 공부하기, 수업 진도

점수 : 40점
1. 리액트 완강 : 절반 들음
2. cs공부하기 : 너무 조금 함
3. 수업 진도 : 따라감

총평 - 할 게 많아지니 불가능한 목표를 세웠는데.. 반성하고 실현 가능한 목표를 세우겠습니다...

**Comment by YoungSeok222222** [comment]
이번주 목표: IM 통과하기, 인적성 공부하기, 진도 따라가기, 블로그 글 3개 이상 올리기

점수:  50점
이유: IM 통과하기(완료), 인적성 공부하기(이제 시작..), 진도 따라가기(아예 안 함), 블로그 글 3개 이상 올리기(완료)

총평 -운 좋게도 IM은 통과하였습니다. 하지만 인적성 공부 때문에 진도를 아예 따라가지 않고 있습니다. 그렇다고 해서 인적성 공부를 맛깔나게 하지도 않았기 때문에(사실 이제 시작해서) 이번주 목표는 아예 인적성만 파도록 할 계획입니다.

**Comment by itmakesmesoft** [comment]
이번 주 목표 : 자바 강의 객체지향 1까지 듣기(제발)

점수 : 50점
이유 : 그 어느때보다 열심히는 했으나, 배열까지밖에 못들음ㅠㅠ
알고리즘 시험 대비 안하려고 하다가 급하게 하게되어서 자바를 못들음..

**Comment by HyeseungNA** [comment]
IM 대비 오답노트 블로그에 정리하기,수업 진도 놓치지 않기
 점수 : 50점
이유: im대비 오답노트는 다 작성하였습니다. 장고 수업진도는 전혀 따라가지 못했습니다ㅠㅠ
